src/backend/joanie/core/templatetags/extra_tags.py

```diff
@@ -115,7 +117,8 @@ def format_currency_with_symbol(value: Money):
     """
     parts = str(value).split()
     amount = parts[0]
-    configuration_active_language = settings.LANGUAGE_CODE
-    activate_language = configuration_active_language.replace("-", "_")
-    locale = activate_language[:-2] + activate_language[-2:].upper()
-    return format_currency(amount, settings.DEFAULT_CURRENCY, locale=locale)
+    return format_currency(
+        amount,
+        settings.DEFAULT_CURRENCY,
+        locale=get_language_settings(get_language()).get("code").replace("-", "_"),
```

Comment:
babel package has a class to parse locale and get a babel `Locale` object.

```python
from babel.core import Locale

language_code = get_language_settings(get_language()).get("code")
locale = Locale(language_code, sep='-')
```

Reply:
Perfect ! thanks :pray: